frontend/src/components/mine/ContactInfo/ViewMineManager.js

```diff
@@ -25,127 +25,180 @@ const propTypes = {
 const defaultProps = {
   mine: {},
 };
- 
+
 export class ViewMineManager extends Component {
-  state = { isHistoryVisible: false, }
+  state = { isHistoryVisible: false };
 
   handleSubmit = (values) => {
-    this.props.handlePartySubmit(values, ModalContent.PERSON)
-  }
+    this.props.handlePartySubmit(values, ModalContent.PERSON);
+  };
   /**
    * change mine manager on record.
    */
   handleManagerSubmit = (values) => {
-    this.props.addMineManager(this.props.mine.guid, values.mineManager, this.props.mine.mine_detail[0].mine_name, values.startDate).then(() => {
-      this.props.fetchMineRecordById(this.props.mine.guid);
-      this.props.closeModal();
-    })
-  }
+    this.props
+      .addMineManager(
+        this.props.mine.guid,
+        values.mineManager,
+        this.props.mine.mine_detail[0].mine_name,
+        values.startDate
+      )
+      .then(() => {
+        this.props.fetchMineRecordById(this.props.mine.guid);
+        this.props.closeModal();
+      });
+  };
 
   toggleMineManagerHistory = () => {
     this.setState({
       isHistoryVisible: !this.state.isHistoryVisible,
-    })
-  }
+    });
+  };
 
   openModal(event, onSubmit, handleChange, handlePartySubmit, title) {
     event.preventDefault();
     this.props.openModal({
-      props: { onSubmit, handleChange, handlePartySubmit, title},
-      content: modalConfig.UPDATE_MINE_MANAGER
+      props: { onSubmit, handleChange, handlePartySubmit, title },
+      content: modalConfig.UPDATE_MINE_MANAGER,
     });
   }
 
   render() {
     const { mine } = this.props;
-      return (
-        <div>
-        {!mine.mgr_appointment[0] &&
-            <Card>
-              <NullScreen 
-                type='manager'
-                />
-              <div className="center">
-                <ConditionalButton 
-                  handleAction={(event) => this.openModal(event, this.handleManagerSubmit, this.props.handleChange, this.handleSubmit, ModalContent.ADD_MINE_MANAGER)} 
-                  string={ModalContent.ADD_MINE_MANAGER}
-                  type="primary"
-                  />
-              </div>
-            </Card>
-          }
-        {mine.mgr_appointment[0] && 
+    return (
+      <div>
+        {!mine.mgr_appointment[0] && (
+          <Card>
+            <NullScreen type="manager" />
+            <div className="center">
+              <ConditionalButton
+                handleAction={(event) =>
+                  this.openModal(
+                    event,
+                    this.handleManagerSubmit,
+                    this.props.handleChange,
+                    this.handleSubmit,
+                    ModalContent.ADD_MINE_MANAGER
+                  )
+                }
+                string={ModalContent.ADD_MINE_MANAGER}
+                type="primary"
+              />
+            </div>
+          </Card>
+        )}
+        {mine.mgr_appointment[0] && (
```

Comment:
Did you decide that this was safe? Or do we need a `mine.mgr_appointment` check?